backend/src/routes/v0/thread.py

```
@@ -223,6 +224,47 @@ async def get_thread(
         ) from e
 
 
+@router.get(
+    "/threads/{thread_id}/stream",
+    name="Stream Thread Results",
+    operation_id="ruska_stream_thread",
+    tags=["Thread"],
+)
+async def stream_thread(
+    thread_id: str,
+    user: ProtectedUser = Depends(get_optional_user_from_token),
```

Ruff (0.14.10)

Unused function argument: `user`

(ARG001)